models/user.go

```diff
@@ -119,6 +121,11 @@ func (u *User) BeforeUpdate() {
 	u.UpdatedUnix = time.Now().Unix()
 }
 
+// Set time to last login
+func (u *User) UpdateLastLogin() {
```

Comment:
@andreynering, @0xbaadf00d asked to pu this name, originally was `afterLogin()`
> Maybe UpdateLastLogin will be a more appropriate name for this function
I will provide fix for this.